### ISSUE TITLE
phpmentoring.org is down

### ISSUE THREAD

**Issue by notisset** [phpmentoring/phpmentoring.github.com#28]
As of this morning (CET), the site shows the dns provider (Gandi.net) default page

**Comment by euneuber** [comment]
site is still offline?!

**Comment by notisset** [comment]
Yes it is still offline. phpmentoring.org gets resolved  to 217.70.184.38 (webredir.vip.gandi.net) by both Google Public DNS and OpenNIC
